openhwgroup/core-v-verif
# TB updates and fixes

This PR updates includes those changes from @dd-vaibhavjain :

- Core hash to latest at the time of PR
- add rvviRefMetricGet to solve core-v-verif issue https://github.com/openhwgroup/core-v-verif/issues/2015
- Add param for trace2api ON_MISMATCH_RECONVERGE
- Update mk/README to add desc for TEST_CFG_FILE

FIX for simd and hwloop test issues:
- fix SIMD UIMM case for update_imm_str
- update imm randomization for cv32e40p_inst type
- Fix constraints issues in hwloop tests for setupi instructions:
- This is done by simplifying constraints and and fix issues with cv.setupi, resereved_rd.
- Add checks to make sure number random generated instructions never underflow due to constraints
